Content.Server/Chemistry/Components/SolutionRegenerationComponent.cs

```diff
@@ -14,31 +14,30 @@ public sealed partial class SolutionRegenerationComponent : Component
     /// <summary>
     /// The name of the solution to add to.
     /// </summary>
-    [DataField("solution", required: true), ViewVariables(VVAccess.ReadWrite)]
+    [DataField("solution", required: true)]
     public string SolutionName = string.Empty;
 
     /// <summary>
     /// The solution to add reagents to.
     /// </summary>
-    [DataField("solutionRef")]
-    public Entity<SolutionComponent>? Solution = null;
+    public Entity<SolutionComponent>? SolutionRef = null;
 
     /// <summary>
     /// The reagent(s) to be regenerated in the solution.
     /// </summary>
-    [DataField("generated", required: true), ViewVariables(VVAccess.ReadWrite)]
+    [DataField("generated", required: true)]
```

Comment:
```suggestion
    [DataField(required: true)]
```
Kill it

Reply:
![science](https://github.com/space-wizards/space-station-14/assets/107586145/7b7201f6-3b89-40bc-83e7-cfa6426de37d)